modules/bpi/bpi.api.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @file
+ * Describe hooks provided by the BPI module.
+ */
+
+/**
+ * @addtogroup hooks
+ * @{
+ */
+
+/**
+ * Return a list of BPI node types a module can handle.
+ *
+ * @return array
+ *   An array of BPI type names.
+ */
+function hook_bpi_syndicate_get_bpi_types() {
+    return array();
+}
+
+/**
+ * Get form for mapping a BPI node to a Drupal content type.
+ *
+ * @param string $bpi_type
+ *   The BPI type.
+ * @param string $content_type
+ *   The Drupal content type.
+ * @param array $field_mapping
+ *   The current field mapping.
+ *
+ * @return array
+ *   The mapping form.
+ */
+function hook_bpi_get_field_mapping_form($bpi_type, $content_type, array $field_mapping) {
+    return array();
+}
+
+/**
+ * Alter the form for defining a BPI node mapping.
+ *
+ * @param string $bpi_type
+ *   The BPI node type.
+ * @param array $form_item
```

Comment:
Is there a reason why this parameter is called `$form_item` but referred to as "form"?

Reply:
No, it's a typo.